packages/framework/src/Model/Product/Search/FilterQueryFactory.php

```diff
@@ -171,9 +170,9 @@ public function createSellableAndVisible(): FilterQuery
      * @param \Shopsys\FrameworkBundle\Model\Product\Filter\ProductFilterData $productFilterData
      * @return \Shopsys\FrameworkBundle\Model\Product\Search\FilterQuery
      */
-    public function createProductFilterCount(int $categoryId, ProductFilterData $productFilterData): FilterQuery
+    public function createListableProductsByCategoryWithPriceAndStockFilter(int $categoryId, ProductFilterData $productFilterData): FilterQuery
```

Comment:
```suggestion
    public function createListableProductsByCategoryIdWithPriceAndStockFilter(int $categoryId, ProductFilterData $productFilterData): FilterQuery
```

Reply:
Fixed in f8fc767559aacbe9030213df62204e387e78a385